sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -48,7 +55,7 @@ def get_acr_access_token(self, challenge, **kwargs):
 
     def exchange_refresh_token_for_access_token(self, refresh_token, service, scope, grant_type, **kwargs):
```

Comment:
This method doesn't seem to be used by anything but this class's `get_acr_access_token` method, which itself doesn't seem to be used (which makes sense, given that this is an anonymous client). Do you know if there's any reason to keep these methods?

Reply:
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_authentication_policy.py#L67